src/services/item/plugins/app/appData/schemas.ts

```diff
@@ -62,20 +74,33 @@ export const updateOne = {
   }),
   response: {
     [StatusCodes.OK]: appDataSchemaRef,
+    '4xx': errorSchemaRef,
   },
 } as const satisfies FastifySchema;
 
 export const deleteOne = {
+  operationId: 'deleteAppData',
+  tags: ['app', 'app-data'],
+  summary: 'Delete app data',
+  description: 'Delete given app data.',
+
   params: Type.Object({
     itemId: customType.UUID(),
     id: customType.UUID(),
   }),
   response: {
-    [StatusCodes.OK]: customType.UUID(),
+    [StatusCodes.OK]: customType.UUID({ descritpion: 'Successful Response' }),
+    '4xx': errorSchemaRef,
   },
 } as const satisfies FastifySchema;
 
 export const getForOne = {
+  operationId: 'getAppDataForApp',
+  tags: ['app', 'app-data'],
+  summary: 'Get all app data of an app',
+  description:
+    'Get app data saved for an app, depending on the permission of the user and the data visibility.',
```

Comment:
We have the possibility to specify a `type` filter to only get app-data of the specified type.

Reply:
I added the info in the description of the qs directly.